migrations/2024-12-17-141811_add_relay_table/up.sql

```diff
@@ -0,0 +1,22 @@
+-- Your SQL goes here
+CREATE TYPE "RelayStatus" AS ENUM ('created', 'pending', 'failure', 'success');
+
+CREATE TYPE "RelayType" AS ENUM ('refund');
```

Comment:
Should we consider using a string instead of enum for the `relay_type` field? This is considering that adding support for new relay types would involve database migrations to add enum variants each time.

This is open for discussion, we need not take it up now.